wikidata/entity.py

```diff
@@ -238,6 +247,14 @@ def attributes(self) -> Mapping[str, object]:
     def load(self) -> None:
```

Comment:
It should return early if `self.state is Entity.non_existent`, because `Entity.non_existent` means the server will respond with nothing anyway.